about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="doc.css" />
+    <link rel="stylesheet" href="main.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <div class="nav-container">
+      <nav class="nav-wrap" id="back">
+        <div class="nav-brand">
+          <h1 class="brand-logo">
+            PopCube <span style="color: #fc452e">Ui</span>
```

Comment:
Set colors using CSS variables rather than giving them as hex values. This will help with readability and maintainablity and avoid inline styling as much as possible.